lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -831,78 +831,141 @@ public static function get_style_variations_from_directory( $directory, $scope =
 		return $variations;
 	}
 
-
-	/**
-	 * Resolves relative paths in theme.json styles to theme absolute paths
-	 * and returns them in an array that can be embedded
-	 * as the value of `_link` object in REST API responses.
-	 *
-	 * @since 6.6.0
-	 * @since 6.7.0 Added support for resolving block styles.
-	 *
-	 * @param WP_Theme_JSON_Gutenberg $theme_json A theme json instance.
-	 * @return array An array of resolved paths.
-	 */
-	public static function get_resolved_theme_uris( $theme_json ) {
-		$resolved_theme_uris = array();
+	private static function process_theme_uris( $theme_json, $options = array() ) {
```

Comment:
Everything here and below is a major refactor. Just testing things out for now. 

Basically so we have a single source of truth about where these types of URIs live.